### PR TITLE
Defect slides not updating

### PR DESCRIPTION
**Pull request by tatwell**
https://trello.com/c/JAQl9XQQ/66-defect-slides-in-signage-aren-t-updating-as-expected

Fixes Issue #142.

### CODE REVIEW

File: app/controllers/signs_controller.rb [hunk 1]
```diff
@@ -4,11 +4,11 @@ class SignsController < ApplicationController
   layout 'admin', except: [:play]
 
   skip_before_action :authenticate_user!, only: [:play, :poll]
-  
+
   before_action :set_sign, only: [:remove_slide, :poll, :show, :edit, :update, :destroy, :play, :settings, :order]
   before_action :set_search_filters, only: [:index]
 
-  load_and_authorize_resource
+  load_and_authorize_resource except: [:play, :poll]
```
Comment (tatwell) [inline app/controllers/signs_controller.rb]: One mystery here if anyone wants to take a stab at explaining it: `play` endpoint worked even without this exception. I added it nonetheless for the sake of clarity. But any idea why? Something to do with CanCan?

Comment (homeofmatt) [inline app/controllers/signs_controller.rb]: huh. I can take a look at it sometime soon when I have a moment of downtime. I'll be in the signage app for a bit anyways for my current task.

**Comment by homeofmatt** [comment]
👍

**Comment by mirabelrice** [comment]
Looks good 👍

**Pull request merged**